resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -33,6 +33,7 @@ if ([string]::IsNullOrEmpty($env:TestRegistryPath)) {
     $global:MagnifierRegistryPath = 'HKCU:\Software\Microsoft\ScreenMagnifier\'
     $global:PointerRegistryPath = 'HKCU:\Control Panel\Cursors\'
     $global:ControlPanelAccessibilityRegistryPath= 'HKCU:\Control Panel\Accessibility\'
+    $global:AudioRegistryPath = 'HKCU:\Software\Microsoft\Multimedia\Audio\'
 }
 else {
     $global:AccessibilityRegistryPath = $global:MagnifierRegistryPath = $global:PointerRegistryPath = $global:ControlPanelAccessibilityRegistryPath = $env:TestRegistryPath

```

Comment:
Don't forget to assign `$global:AudioRegistryPath` with the `$env:TestRegistryPath` so that it can work properly in the pester tests.